wpilibj/src/main/java/edu/wpi/first/wpilibj/smartdashboard/SmartDashboard.java

```diff
@@ -165,45 +174,6 @@ public static boolean isPersistent(String key) {
     return getEntry(key).isPersistent();
   }
 
-  /**
-   * Sets flags on the specified key in this table. The key can not be null.
-   *
-   * @param key the key name
-   * @param flags the flags to set (bitmask)
-   */
-  public static void setFlags(String key, int flags) {
```

Comment:
There are backwards compatibility shims in NT level APIs, so we could keep and deprecate this instead of removing it.

Reply:
Especially since [the flags aren't documented](https://github.com/wpilibsuite/allwpilib/issues/3964), I doubt it's seeing much use. No need for deprecation imo.